addons/medical_treatment/functions/fnc_useItem.sqf

```diff
@@ -29,19 +29,30 @@ scopeName "Main";
 private _useOrder = [[_patient, _medic], [_medic, _patient], [_medic]] select GVAR(allowSharedEquipment);
 
 {
-    private _unit      = _x;
+    private _unit = _x;
     private _unitVehicle = objectParent _unit;
-    private _unitItems = _x call EFUNC(common,uniqueItems);
+    private _unitItems = [_x, 0] call EFUNC(common,uniqueItems);
+    private _unitMagazines = [_x, 2] call EFUNC(common,uniqueItems);
 
     {
-        if (!isNull _unitVehicle && {_x in (itemCargo _unitVehicle)}) then {
-            _unitVehicle addItemCargoGlobal [_x, -1];
-            [_unit, _x] breakOut "Main";
+        if (!isNull _unitVehicle) then {
+            if (_x in (itemCargo _unitVehicle)) then {
+                _unitVehicle addItemCargoGlobal [_x, -1];
+                [_unit, _x] breakOut "Main";
+            };
+            if (_x in ((magazinesAmmoCargo _unitVehicle) apply { _x select 0 })) then {
```

Comment:
We should probably skip magazines with ammo count of 0 here. They'll get cleaned up pretty aggressively by the game already, and adjustMagazineAmmo shouldn't readd an empty mag, but it'd also be good to not display those magazines as available for treatment.

Reply:
Does `magazineCargo` return magazines with 0 ammo?

Reply:
Yes.

Reply:
I think we should skip those, as you said above, but for that we'd need to revert to `magazinesAmmoCargo`. I'm not sure if it's worth it.